GitCommands/CommitDataManager.cs

```diff
@@ -146,35 +148,32 @@ private bool TryGetCommitLog(string commitId, string format, [NotNullWhen(return
             return true;
         }
 
-        private static string ProcessDiffNotes(int startIndex, string[] lines)
+        private static string ProcessDiffNotes(int startIndex, string[] lines, out string? notes)
         {
             int endIndex = lines.Length - 1;
-            if (lines[endIndex] == "Notes:")
+            if (string.Equals(lines[endIndex], RevisionReader.NotesPrefix, StringComparison.Ordinal))
             {
                 // No Notes, ignore
                 endIndex--;
             }
 
             StringBuilder message = new();
-            bool notesStart = false;
+            StringBuilder? notesSb = null;
```

Comment:
As I said before, this is not a good name.
```suggestion
            StringBuilder? notes = null;
```

Reply:
.. and doesn`t need to be anymore due to tuple return type.